core/bindings.rs

```diff
@@ -937,6 +937,7 @@ fn decode(
 
 struct SerializeDeserialize<'a> {
   host_objects: Option<v8::Local<'a, v8::Array>>,
+  cb: Option<v8::Local<'a, v8::Function>>,
```

Comment:
This field could use a more descriptive name.

Reply:
Maybe something like `error_callback`?

Reply:
That works.

Reply:
Alright, thanks! I'll rename it.